### PR TITLE
Add platform admin submission pages

### PR DESCRIPTION
**Pull request by samuelhwilliams**
## 🎫 Ticket
BAU

## 📝 Description
Adds a 'Submissions' tab to the admin panel to list/filter submissions and then see their status + event timeline, to help with any support investigations that may be needed.

## 📸 Show the thing (screenshots, gifs)
image

### Review Readiness
- [x] PR title and description provide sufficient context for reviewers
- [x] Commits are logical, self-contained, and have clear descriptions

### Testing
- [x] I have tested this change and it meets the acceptance criteria for the ticket
- [x] I need the reviewer(s) to pull and run this change locally
- [x] New (non-developer) functionality has appropriate unit and integration tests
- [ ] End-to-end tests have been updated (if applicable)
- [ ] Edge cases and error conditions are tested

### CODE REVIEW

File: app/deliver_grant_funding/admin/entities.py [hunk 1]
```diff
@@ -529,3 +531,82 @@ def _apply_search(self, query, count_query, joins, count_joins, search):
...
+            SubmissionEventType.FORM_RUNNER_FORM_RESET_TO_IN_PROGRESS,
+            SubmissionEventType.FORM_RUNNER_FORM_RESET_BY_CERTIFIER,
+        }:
+            description = form_titles.get(event.related_entity_id, "(form no longer exists)")
+        elif event.event_type == SubmissionEventType.SUBMISSION_DECLINED_BY_CERTIFIER:
+            description = event.data.get("declined_reason") or ""
+
+        items.append(
+            MoJTimelineEvent(
+                title=str(event.event_type),
```
[the remark this answers is not in the document]
Comment (samuelhwilliams) [reply app/deliver_grant_funding/admin/entities.py]: this is wrong

File: app/deliver_grant_funding/admin/entities.py [hunk 2]
```diff
@@ -529,3 +531,84 @@ def _apply_search(self, query, count_query, joins, count_joins, search):
...
+        actor = event.created_by
+        title = str(event.event_type)
+        description = ""
+
+        if event.event_type in {
+            SubmissionEventType.FORM_RUNNER_FORM_COMPLETED,
+            SubmissionEventType.FORM_RUNNER_FORM_RESET_TO_IN_PROGRESS,
+            SubmissionEventType.FORM_RUNNER_FORM_RESET_BY_CERTIFIER,
+        }:
+            title += f": {form_titles.get(event.related_entity_id, '(form no longer exists)')}"
```
Comment (sfount) [inline app/deliver_grant_funding/admin/entities.py:553]: Nice yeah I think leaving the `description` field to stand out where there has been something custom about that event (like a reason for declining or an assessors comment or change request reason)

File: app/assets/admin.scss [hunk 3]
```diff
@@ -2,10 +2,12 @@
 // of Access grant funding or Deliver grant funding.
 
 @use "sass:color";
-@use "govuk/base" as govuk with (
+@use "govuk" as govuk with (
     $govuk-brand-colour: #912b88
 );
 
+@use "components/metadata";
```
Comment (sfount) [inline app/assets/admin.scss:9]: Smallest possible tweak 🤏 I think if we bring in `_tweaks.scss` we'll also get `app-status-tag` which stops the tag overflowing if its got `(Overdue)` or possibly other status amends on

**Review by sfount — APPROVED** [review]
Nice this is going to be really helpful for looking into whats happened on support 👌 I've pulled it locally and tried it out and its looking good (I presume because govuk flask admin pulls in all MOJ sass the timeline styles have been included all along)

File: app/assets/admin.scss (hunk 3 above)
Comment (samuelhwilliams) [reply app/assets/admin.scss:9]: niiiiice catch thank you

**Review by sfount — APPROVED** [review]
